### PR TITLE
Add FinOps Practice Operations doc

### PR DESCRIPTION
**Pull request by arthurclares**
## 🛠️ Description
This small PR adds the guidance for the FinOps Practice Operations capability.

## 📷 Screenshots
NA

## 📋 Checklist
NA

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [x] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Yes (required for `dev` PRs)
> - [x] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [x] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### CODE REVIEW

File: docs/_docs/framework/capabilities/manage/practice-ops.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+grand_parent: FinOps Framework
+parent: Manage
+title: FinOps Practice Operations
+permalink: /framework/capabilities/manage/practice-ops
+nav_order: zzz
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/practice-ops.md:7]: Should we follow the order of the FinOps Foundation? 

For the Manage the FinOps Practice domain the have the capabilities in the following order: 

1. FinOps Practice Operations
2. Cloud Policy & Governance
3. FinOps Assessment 
4. FinOps Tools & Services
5. FinOps Education & Enablement
6. Invoicing & Chargeback 
7. Onboarding Workloads
8. Intersecting Disciplines

File: docs/_docs/framework/capabilities/manage/practice-ops.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+<br>
+
+## 🏗️ Building on the basics
+
+- Form a dedicated FinOps team with diverse skills and adequate resources.
+- Provide continuous [training and skill development](docs\_docs\framework\capabilities\manage\education.md) for team members.
+- Regularly review and refine the FinOps strategy to ensure alignment with organizational objectives.
+- Foster a culture of collaboration and communication across departments to enhance FinOps practices.
+
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/practice-ops.md:62]: Should we reference the FinOps Review assessment so the FinOps team can track progression? 

```suggestion
Leverage the [Microsoft FinOps Review](https://aka.ms/finops/review) to quantify progress.
```

File: docs/_docs/framework/capabilities/manage/practice-ops.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+<br>
+
+## ⏩ Next steps
+
+Related FinOps capabilities:
+
+- [Assessment](../manage/assessment.md)
+- [Benchmarking](../optimize/benchmarking.md)
+
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/manage/practice-ops.md:78]: For related capabilities, should we add FinOps Tools & Services and FinOps Education & Enablement as referenced in the Inputs and Outputs section of the FinOps Foundation/ FinOps Practice Operations Capability https://www.finops.org/framework/capabilities/finops-practice-operations/. 

```suggestion
- [FinOps Tools & Services](docs/_docs/framework/capabilities/manage/education.md)
- [FinOps Education & Enablement](docs/_docs/framework/capabilities/manage/tools-and-services.md)
```

Note: Please verify links if suggestion is added into document.

File: docs/_docs/framework/capabilities/manage/practice-ops.md (hunk 2 above)
Comment (flanakin) [reply docs/_docs/framework/capabilities/manage/practice-ops.md:62]: I added a reference to the assessment capability, which should cover us.

File: docs/_docs/framework/capabilities/manage/practice-ops.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+<br>
+
+## 🏗️ Building on the basics
+
+- Form a dedicated FinOps team with diverse skills and adequate resources.
+- Provide continuous [training and skill development](docs\_docs\framework\capabilities\manage\education.md) for team members.
+- Regularly review and refine the FinOps strategy to ensure alignment with organizational objectives.
+- Foster a culture of collaboration and communication across departments to enhance FinOps practices.
+
+
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/manage/practice-ops.md:63]:
```suggestion
```

File: docs/_docs/framework/capabilities/manage/practice-ops.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+This capability is part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [FinOps Practice and Operations](https://www.finops.org/framework/capabilities/finops-practice-operations/) article in the FinOps Framework documentation.
+
+<br>
+
+## ⏩ Next steps
+
+Related FinOps capabilities:
+
+- [Assessment](../manage/assessment.md)
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/manage/practice-ops.md:76]:
```suggestion
- [FinOps assessment](./assessment.md)
```

File: docs/_docs/framework/capabilities/manage/practice-ops.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+This article helps you understand the FinOps Practice and Operations capability within the FinOps Framework and how to implement it in your organization.
+{: .fs-6 .fw-300 }
+
+<details open markdown="1">
+  <summary class="fs-2 text-uppercase">On this page</summary>
+
+
+</details>
+
+---
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/manage/practice-ops.md:115]:
```suggestion
```

File: docs/_docs/framework/capabilities/manage/practice-ops.md (hunk 3 above)
Comment (flanakin) [reply docs/_docs/framework/capabilities/manage/practice-ops.md:78]: Education and enablement makes sense. I'm not aware of any dedicated tools for FinOps practice operations, so we can probably leave that one out for now. We can add it later if there is something specific that I'm missing.
```suggestion
- [FinOps education and enablement](./education.md)

```

**Review by flanakin — APPROVED** [review]

**Pull request merged**